### DISCUSSION TITLE
16 ottobre 2023 - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#48] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/a9cdae2a-040a-4a30-a9a2-713ef8ed4641)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/c36237d9-f307-4af6-9f54-4c9746043c21)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/5af2bbf9-1a3a-4876-b270-a68d4110007f)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/48cd80c7-bd26-469e-80b0-186fc9929974)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/97389762/919324b4-0d1f-4f66-826c-940e2acf584d)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/97389762/b173189c-0ded-40be-ae31-69e29d02616e)
Credo ci sia un errore nel calcolo della copertura minimale.

**Reply by Alby7503** [reply]
È davvero sufficiente dire che p contiene una superchiave per definirlo senza perdita?

**Reply by CasuFrost** [reply]
Si.

**Reply by dadocimma** [reply]
Si,c’è un BH—>E in più bella copertura minimale.
